bigtable_input.py

```diff
@@ -239,28 +269,38 @@ def parse(r):
         return sorted([parse(r) for r in rows], key=operator.itemgetter(0))
 
     def delete_row_range(self, format_str, start_game, end_game):
+        global _pool
+        if _pool is None:
+            _pool = Pool(16)
```

Comment:
I'd avoid the global pool and just recreate every time (as this presumably is taking a long time)

Reply:
It's true that each call takes awhile (about a minute) so that recreating the pool is not too impactful. I did like the idea of reusing the same pool for an entire trim operation (since delete_row_range is called multiple times through that) but I can refactor it so that the pool is passed in and limited in scope to the trim.

Reply:
Thanks for the prompt on this! I ended up using a properly scoped `Pool` via the `with` statement, and along the way, set the pool size to be dependent on the number of vCPUs.